### PR TITLE
Cleanup pull request #7

### PR DESCRIPTION
**Pull request by pascaldekloe**

### CODE REVIEW

**Comment by pascaldekloe** [comment]
@bnkamalesh ?

File: main.go [hunk 1]
```diff
@@ -158,20 +164,36 @@ func HandlerRepoSVG(w http.ResponseWriter, r *http.Request) {
...
 	}
 
-	badgeName := fmt.Sprintf("%s%s%%s", color, badgeStyle, obj.Cover)
-	badgeSVG, err := getBadgeCache(badgeName)
-	if err == nil && len(badgeSVG) > 0 {
-		serveBadge(w, badgeSVG)
+	badgeName := fmt.Sprintf("%s%s%s", color, badgeStyle, obj.Cover)
+	svg, err := redisRing.Get(badgeName).Bytes()
+	switch err {
+	case nil:
```
Comment (golangci) [inline main.go]: expected ';', found 'if'

File: main.go [hunk 2]
```diff
@@ -158,20 +164,31 @@ func HandlerRepoSVG(w http.ResponseWriter, r *http.Request) {
...
-	badgeName := fmt.Sprintf("%s%s%%s", color, badgeStyle, obj.Cover)
-	badgeSVG, err := getBadgeCache(badgeName)
-	if err == nil && len(badgeSVG) > 0 {
-		serveBadge(w, badgeSVG)
-		return
-	}
+	badgeName := fmt.Sprintf("%s%s%s", color, badgeStyle, obj.Cover)
+	svg, err := redisRing.Get(badgeName).Bytes()
+	if err != nil {
+	 	if err != redis.Nil {
```
Comment (golangci) [inline main.go]: File is not gofmt-ed with -s

**Comment by bnkamalesh** [comment]
I didn't think it was an overkill. Since main has everything, including the redis connectors and such.

Edit: also don't you think it's harder to write tests now that all the functions are combined?

**Comment by pascaldekloe** [comment]
@bnkamalesh, please have a look at the bugs fixed.

**Pull request merged**